packages/@react-stately/layout/src/ListLayout.ts

```diff
@@ -52,6 +53,7 @@ export class ListLayout<T> extends Layout<Node<T>> implements KeyboardDelegate {
   protected headingHeight: number;
   protected estimatedHeadingHeight: number;
   protected padding: number;
+  protected defaultNoheaderCellWidth: number;
```

Comment:
had a chat with @devongovett and we came up with an alternative

this isn't really applicable to ListLayout, can we move it to a new constructor in TableLayout?
It should call super

Lets also change the option to be a function `getDefaultWidth(column)`

This will allow us to move default width out of stately and up to the react-spectrum package since it's spectrum specific